cmd/cody-gateway/internal/actor/productsubscription/productsubscription.go

```diff
@@ -10,11 +10,12 @@ import (
 	"github.com/Khan/genqlient/graphql"
 	"github.com/gregjones/httpcache"
 	"github.com/sourcegraph/log"
-	"github.com/sourcegraph/sourcegraph/internal/collections"
 	"github.com/vektah/gqlparser/v2/gqlerror"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/trace"
 
+	"github.com/sourcegraph/sourcegraph/internal/collections"
+
```

Comment:
```suggestion
	"github.com/sourcegraph/sourcegraph/internal/collections"
```

Actually needs a import resort.

Reply:
Fixed!